addons/api/mirage/config.js

```diff
@@ -23,13 +23,16 @@ export default function() {
 
   // Scope resources
 
-  this.get('/scopes', ({ scopes }, { queryParams: { scope_id } }) => {
+  this.get('/scopes', ({ scopes }, { queryParams: { scope_id, recursive } }) => {
     // Default parent scope is global
     if (!scope_id) scope_id = 'global';
-    const results = scopes.where(scope => {
-      return scope.scope ? scope.scope.id === scope_id : false
-    });
-    return results;
+    if (recursive && scope_id === 'global') {
+      return scopes.all();
```

Comment:
This may not be exactly right.  @jefferai, if recursive scopes are queried with ID `global`, it should return all scopes _except_ global, correct?  And if scopes are queried _with only_ recursive (no ID), would all scopes including global be returned?

Reply:
Nope, recursion _includes_ the scope being queried. If you aren't recursing, we'd show scopes where the parent is the one given, but when recursing scopes from global we include global as well, because otherwise there'd be no way to include it (since it doesn't have a parent).